cmd/chisel/cmd_cut.go

```diff
@@ -80,13 +93,21 @@ func (cmd *cmdCut) Execute(args []string) error {
 		return err
 	}
 
+	arch := cmd.Arch
+	if arch == "" {
+		arch := goToDebArch[runtime.GOARCH]
+		if arch == "" {
+			return fmt.Errorf("cannot infer architecture from current platform %s, see the --arch option", runtime.GOARCH)
```

Comment:
Can we please have this as `deb.InferArch() (string, error)`, with some testing?

Reply:
Added `deb.ValidateArch() (string, error)` to `internal/deb/helpers.go` which validates package architecture if given non-empty string or infers one from current platform. Tested in `internal/deb/helpers_test.go`.